posts/inside-rust/2020-03-12-twir-new-lead.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: "Vikrant (nasa42) is retiring from This Week in Rust. TWiR is looking for a new maintainer."
+author: <TBD>
```

Comment:
I don't mind being the author for the sake for the post, but if someone else is more appropriate, they can take it.
```suggestion
author: Erin Power
```